// Author of  EPSC-LOAM : QZ Wang
// Email wangqingzhi27@outlook.com

#ifndef _DISTORTION_ADJUST_H_
#define _DISTORTION_ADJUST_H_

#include "utility.h"
#include "common.h"

class DistortionAdjust
{
private:
	



public:
	DistortionAdjust();
	~DistortionAdjust();
};


#endif  // _DISTORTION_ADJUST_H_